CMakeLists.txt

```
@@ -18,6 +18,8 @@ set(PACKAGE_VERSION_MAJOR "0")
 set(PACKAGE_VERSION_MINOR "1")
 set(PACKAGE_VERSION_PATCH "0")
 
+set(THIRD_PARTY_DIR "${PROJECT_SOURCE_DIR}/third-party")
+
```

I believe it should be without quotes:
```suggestion
set(THIRD_PARTY_DIR ${PROJECT_SOURCE_DIR}/third-party)

```
My understanding is with quotes CMake interprets it as a string, without it is interpreted as a path.